src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/VisibleMemberTable.java

```diff
@@ -610,11 +691,12 @@ boolean allowInheritedMethod(ExecutableElement inheritedMethod,
         Elements elementUtils = config.docEnv.getElementUtils();
 
         // Check the local methods in this type.
+        // List contains overloads and probably something else, but one match is enough, hence short-circuiting
         List<Element> lMethods = lmt.getMembers(inheritedMethod, Kind.METHODS);
         for (Element le : lMethods) {
             ExecutableElement lMethod = (ExecutableElement) le;
             // Ignore private methods or those methods marked with
-            // a "hidden" tag.
+            // a "hidden" tag. // FIXME I cannot see where @hidden is ignored
```

Comment:
Generally, there is `utils.hasHiddenTag`

It _may_ be a good design question as to when `@hidden` should be taken into account.  For example, should VMT build an accurate model and only suppress on output, or should `@hidden` be reflected in the VMT model.

If it helps (?) I note that at one time a precursor of `@hidden` was `@treatAsPrivate`